grakn-migration/json/src/main/java/ai/grakn/migration/json/JsonMigrator.java

```diff
@@ -46,28 +46,30 @@
     private final Set<Reader> readers;
 
     public static void main(String[] args) {
-        MigrationCLI.init(args, JsonMigrationOptions::new).stream()
-                .filter(Optional::isPresent)
-                .map(Optional::get)
-                .forEach(JsonMigrator::runJson);
+        try {
+            MigrationCLI.init(args, JsonMigrationOptions::new).stream()
+                    .filter(Optional::isPresent)
+                    .map(Optional::get)
+                    .forEach(JsonMigrator::runJson);
+        } catch (Throwable throwable){
+            MigrationCLI.die(throwable.getMessage());
```

Comment:
Ah okay. So new question, why not just inline it and do `System.err.println(errorMsg);` or are we planning for more handling to go into that method? 

One line methods which only print stuff should usually be inlined.